File: src/entitie/comment.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entitie;

/**
 *
 * @author LENOVO-PC
 */
public class comment {

private int idc;
private String textec;
private String datec;
private String id;

    public void setIdc(int idc) {
        this.idc = idc;
    }

    public void setTextec(String textec) {
        this.textec = textec;
    }

    public void setDatec(String datec) {
        this.datec = datec;
    }

    public void setId(String id) {
        this.id = id;
    }

    public int getIdc() {
        return idc;
    }

    public String getTextec() {
        return textec;
    }

    public String getDatec() {
        return datec;
    }

    public String getId() {
        return id;
    }

    public comment(String textec, String datec) {
        
        this.textec = textec;
        this.datec = datec;
        
    }
    public comment(String textec, String datec, String id) {
        
        this.textec = textec;
        this.datec = datec;
        this.id = id;
        
    }
    
}

File: PIDEVFrogs/src/JardinBundle/Entity/Comment.php
<?php

namespace JardinBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Comment
 *
 * @ORM\Table(name="comment")
 * @ORM\Entity
 */
class Comment
{
    /**
     * @var integer
     *
     * @ORM\Column(name="idc", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idc;

    /**
     * @var string
     *
     * @ORM\Column(name="textec", type="string", length=50, nullable=false)
     */
    private $textec;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="datec", type="date", nullable=false)
     */
    private $datec;

    /**
     * @var string
     *
     * @ORM\Column(name="id", type="string", length=88, nullable=false)
     */
    private $id;



    /**
     * Get idc
     *
     * @return integer
     */
    public function getIdc()
    {
        return $this->idc;
    }

    /**
     * Set textec
     *
     * @param string $textec
     *
     * @return Comment
     */
    public function setTextec($textec)
    {
        $this->textec = $textec;

        return $this;
    }

    /**
     * Get textec
     *
     * @return string
     */
    public function getTextec()
    {
        return $this->textec;
    }

    /**
     * Set datec
     *
     * @param \DateTime $datec
     *
     * @return Comment
     */
    public function setDatec($datec)
    {
        $this->datec = $datec;

        return $this;
    }

    /**
     * Get datec
     *
     * @return \DateTime
     */
    public function getDatec()
    {
        return $this->datec;
    }

    /**
     * Set id
     *
     * @param string $id
     *
     * @return Comment
     */
    public function setId($id)
    {
        $this->id = $id;

        return $this;
    }

    /**
     * Get id
     *
     * @return string
     */
    public function getId()
    {
        return $this->id;
    }
}
